myscript.js

```diff
@@ -0,0 +1,21 @@
+
+var dataArray = [1, 8, 7, [ 4, 65, 1, [ 2, 7 ], 45, [ 6, 4], 6 ], 4, 8]
+
+var jenyaFlatternArray = function (Array) {
```

Comment:
sure, I think this is very good guideline: https://github.com/airbnb/javascript
some things might not be clear for you, but stick to the rules that you can understand for now